src/ui/public/assets/js/utils.js

```diff
@@ -111,7 +111,7 @@ function setMouseOverElementContent($elem, content) {
         "data-toggle": "popover",
         "data-placement": "right",
         "data-content": content
-    }).popover({ trigger: "manual", animation: false });
+    }).popover({trigger: "manual", animation: false});
```

Comment:
<nitpick> Can you please use the same coding style as everywhere else? I.e. here

```
}).popover({ trigger: "manual", animation: false });
```